### PR TITLE
denso_robot_ros: 3.1.1-1 in 'melodic/distribution.yaml' [bloom]

### PR DESCRIPTION
**Pull request by furukawahiromi**
Increasing version of package(s) in repository `denso_robot_ros` to `3.1.1-1`:

- upstream repository: https://github.com/DENSORobot/denso_robot_ros.git
- release repository: https://github.com/DENSORobot/denso_robot_ros-release.git
- distro file: `melodic/distribution.yaml`
- bloom version: `0.10.1`
- previous version for package: `null`

## bcap_core

```
* Change the symbolic link to the directory of include/bcap_core
```

## bcap_service

- No changes

## bcap_service_test

- No changes

## denso_robot_bringup

- No changes

## denso_robot_control

```
* Add ros::spinOnce() for the callbacks.
```

## denso_robot_core

- No changes

## denso_robot_core_test

- No changes

## denso_robot_descriptions

```
* Fix Controller ABORTED ([#5](https://github.com/DENSORobot/denso_robot_ros/issues/5))
```

## denso_robot_gazebo

- No changes

## denso_robot_moveit_config

- No changes

## denso_robot_ros

- No changes

### CODE REVIEW

**Comment by nuclearsandwich** [comment]
Holding for Melodic [sync](https://discourse.ros.org/t/preparing-for-melodic-sync-2021-03-15/19458).

File: melodic/distribution.yaml
Comment (clalancette) [inline melodic/distribution.yaml:2144]: There's no LICENSE file at the top-level of the repository.  While it isn't technically required, it is a good idea to have one at the top for people looking at your repository.

My suggestion is to add a LICENSE file with the MIT license in it.  Just put a comment here once you've done that.  No need to do another bloom release afterwards; it won't change the state of the release artifacts.

**Review by clalancette — CHANGES_REQUESTED** [review]

File: melodic/distribution.yaml
Comment (furukawahiromi) [reply melodic/distribution.yaml:2144]: Thank you for the suggestion. I've done it.

Comment (mabelzhang) [reply melodic/distribution.yaml:2144]: I'm not seeing the added file. Did you add it to the source repo? I'm looking here https://github.com/DENSORobot/denso_robot_ros and there's no LICENSE file. I only see a `License.txt` in `bcap_core` dated 3 years ago with the MIT license, is that by chance the file added? I think the comment above means adding a file named LICENSE to the root of the repo.

Comment (furukawahiromi) [reply melodic/distribution.yaml:2144]: I added to [melodic-devel](https://github.com/DENSORobot/denso_robot_ros/tree/melodic-devel) branch. I'll merge master branch after this merge.

**Comment by jacobperron** [comment]
LGTM, thanks for adding the LICENSE file!

**Pull request merged**
